code/io/key.cpp

```diff
@@ -54,7 +54,8 @@ SDL_mutex* key_lock;
 								// when typing in your pilots callsign.  This global flag is checked before execution
 								// is stopped.
 
-SCP_map<int, int> SDLtoFS2;
+SCP_map<SDL_Scancode, int> SDLtoFS2;
+SCP_map<int, SDL_Scancode> FS2toSDL; // Convenience map to speed up FS2->SDL lookups needed for displaying localized keycodes in config
```

Comment:
Consider using only the string -> int map. To do reverse map lookups you can perform a seach on the ::second member of the key-value pair.

We can do this because both the key and the values are both unique to each other, and int searches are far simpler to check for equality than it is to check strings.

Reply:
SDL_Scancode is an Enum, so techincally just an int compare.
I did this to avoid using extra search time when searching the map by value, but if you want to keep it neater and not clutter code and memory with this reverse map, I can change it.

Reply:
Yes please.  I recommend making a function with the reverse map's name to do the job.